app/Dockerfile

```diff
@@ -49,6 +48,13 @@ RUN set -eux \
     && pip install azure-cli==${AZURE_CLI_VERSION} \
     && az --version
 
+###
+### Ensure AWS cli version is present and validated
+###
+RUN set -eux \
+    && apk add --no-cache aws-cli \
```

Comment:
Please move it to line `27` - the is no point to create a new layer with only `awscli` IMO

Reply:
Done